src/openrct2/CMakeLists.txt

```diff
@@ -101,17 +101,9 @@ if (NOT DISABLE_GOOGLE_BENCHMARK)
 endif ()
 
 if (ENABLE_SCRIPTING)
-    if (MSVC)
-        find_package(duktape REQUIRED)
-    else ()
-        PKG_CHECK_MODULES(DUKTAPE REQUIRED duktape)
-        target_include_directories(${PROJECT_NAME} PRIVATE ${DUKTAPE_INCLUDE_DIRS})
-        if (STATIC)
-            target_link_libraries(${PROJECT_NAME} ${DUKTAPE_STATIC_LIBRARIES})
-        else ()
-            target_link_libraries(${PROJECT_NAME} ${DUKTAPE_LIBRARIES})
-        endif ()
-    endif ()
+    find_package(duktape CONFIG REQUIRED)
```

Comment:
… when it can still require duktape here?

Reply:
vcpkg provides the same duktapeConfig.cmake file in its ports.